### DISCUSSION TITLE
Broken Package. Where to request update at?

### DISCUSSION THREAD

**Discussion by NoVaSpikeXX** [linuxmint/discussions#608] [Questions] [answered]
Pysolfc is a broken package. I just installed mint for some older people to go online and play games, its always solitaire somehow; but Pysolfc does not work.

Package: https://community.linuxmint.com/software/view/pysolfc

I went on the developer's website and he tells us to update the package to get it to work: https://pysolfc.sourceforge.io/faq.html

Where do I submit an update request?

**Comment by rcalixte** [comment, marked as answer]
> Where do I submit an update request?

This request would normally go upstream to Ubuntu's bug tracker but this is an unusual situation. Canonical/Ubuntu opted to release Ubuntu 24.04 with Python 3.12, ahead of Debian migrating the package. This has led to a situation where the `pysolfc` package will not work until it is updated. The main issue with getting it updated is that it would require being backported which is highly unlikely at this point. On top of that, Mint 22.x will stay on the same Ubuntu base repository for packages until Mint 23.x.

Without going into too many (more) details, your simplest option would be to install the [Flatpak version of PySolFC from Flathub](https://flathub.org/apps/io.sourceforge.pysolfc.PySolFC). You could remove this and revert to the system package with the release of Mint 23.0, assuming the situation has improved or you can stay on Mint 21.3 until Mint 23.0 is released some time in 2026 (21.3 will receive security updates until April 2027).

Hopefully, this conflict is resolved upstream by the next major release.

**Comment by Fantu** [comment]
If the package is unusable like it was on Debian before latest upload ( https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1008708 ), I think is possible to try to request to backport the latest version as stable update: https://canonical-sru-docs.readthedocs-hosted.com/en/latest/reference/requirements/#what-is-acceptable-to-sru
If not accepted for a stable update, can require at least to add in backports.
